### DISCUSSION TITLE
Benchmarking?

### DISCUSSION THREAD

**Discussion by Kommynct** [anyrun-org/anyrun#187] [Q&A]
I currently use https://github.com/philj56/tofi/issues because it has unmatched performance

Has anyone done any benchmarking for this? I haven't found any and time doesn't work since I have to hit esc

Plus I'm willing to bet the font loading feature makes it faster

**Comment by abenz1267** [comment]
... does it matter? Generally yes, launching a fresh GTK instance will be slower than something custom/non-gtk. GTK isn't the fastest. GTK has mechanism to mitigate that though. Anyrun does not implement those and it'd require a large amount of refactoring i think.

For reference: on my system (!!!) Walker (another gtk-based 'launcher') is AT MOST 1 frame (144hz) slower than tofi. That's basically irrelevant.

In general i'd simply advise using something that you are happy with and not based on benchmarks, which might be irrelevant at best.

**Reply by Kommynct** [reply]
To me, benchmarks are literally the only thing that matter for a launcher, since the features are almost always completely identical.
